routes/api-routes.js

```diff
@@ -32,59 +35,69 @@ module.exports = (app) => {
             }
         });
     });
-    // GET route for getting all of the posts
-    app.get("/api/posts/", (req, res) => {
-        db.Post.findAll({}).then((dbPost) => res.json(dbPost));
-    });
+ 
 
-    // Get route for returning posts of a specific source
-    app.get("/api/posts/source/:source", (req, res) => {
-        db.Post.findAll({
-            // GET * FROM posts WHERE source = req.params.source
-            where: {
-                source: req.params.source,
-            },
-        }).then((dbPost) => {
-            res.json(dbPost);
-        });
-    });
 
-    // Get route for retrieving a single review
-    app.get("/api/posts/:id", (req, res) => {
-        db.Post.findOne({
-            where: {
-                id: req.params.id,
-            },
-        }).then((dbPost) => res.json(dbPost));
-    });
+// // Requiring our Todo model
+// const db = require('../models');
 
-    // POST route for saving a new review
-    app.post("/api/posts", (req, res) => {
-        console.log(req.body);
-        db.Post.create({
-            title: req.body.title,
-            review: req.body.review,
-            rating: req.body.rating,
-            source: req.body.source,
-            author: req.body.author,
-        }).then((dbPost) => res.json(dbPost));
-    });
+// // Routes
+// // =============================================================
+// module.exports = (app) => {
+  // GET route for getting all of the posts
+  app.get('/api/reviews/', (req, res) => {  // same format must be lower case, plural in api/ path 
+    db.Review.findAll({}).then((dbReview) => res.json(dbReview)); //Review is same format as variable Post in review.js in original excercise which was Post (capital P and singular)
```

Comment:
Don't put this in the code, this means nothing to someone reading your code after this pull request so just leave it as a comment in the pull request

Reply:
Do you mean the //  comments not the app.get call?

Reply:
Yup